SIRO_README.md

```diff
@@ -22,22 +22,34 @@ Project-specific README for SIRo.
 
 # Floorplanner Dataset
 
-FP is actually four distinct pieces: Floorplanner scenes, Amazon-Berkeley objects, Google Scanned objects, and Floorplanner episodes.
+FP is actually five distinct pieces: (1) Floorplanner scenes, (2) Amazon-Berkeley objects, (3) Google Scanned objects, (4) Floorplanner scenes, and (5) Floorplanner episodes. (assuming you already have ycb objects)
 
-1. Download Floorplanner Scenes: [fpss_osmm.zip](https://drive.google.com/file/d/1-utUMfUbbzg_zUE5GcGNdk1UEK6lSbXe/view?usp=sharing)
+1. Download Floorplanner Scenes:
+```
+# Go to the habitat data directory
+cd habitat-lab/data
+# Make sure you have git-lfs installed
+git lfs install
+# Clone dataset, and it will take a while to finish
+git clone https://huggingface.co/datasets/fpss/fphab
+# change the name, and Floorplanner Scenes is ready to use
+mv fphab fpss
+```
 2. Download [Amazon and Google object archives](https://drive.google.com/drive/folders/1x6i3sDYheCWoi59lv27ZyPG4Ii2GhEZB)
-3. Download FP episodes: [floorplanner.zip](https://drive.google.com/file/d/1Guxn6v2SC5kAtouwMs1DkBJMK3WW0die/view?usp=sharing)
-4. Extract these into `habitat-lab/data` as follows:
+3. Extract these two object datasets into `habitat-lab/data` as follows:
 ```
-cd data
-unzip ~/Downloads/fpss_osmm.zip
-mv fpss_osmm fpss
 cd objects
 tar -xvf ~/Downloads/google_object_dataset.tar.gz
 tar -xvf ~/Downloads/amazon_berkeley.tar.gz
-cd ../datasets
-unzip ~/Downloads/floorplanner.zip
 ```
+4. Download FP episodes:
+```
+# Go to the habitat datasets directory
+cd habitat-lab/data/datasets
+# Clone dataset
+git clone https://github.com/jimmytyyang/floorplanner.git
```

Comment:
Also, how about a public repo for this data? I don't think it's very sensitive data, and it'll reduce headaches for new users that don't have to bug you for repo access.

Reply:
Hi Eric! thanks for this! I am not sure if this is okay as I think it is not legal approved yet

Reply:
the choice of the name is to reduce the burden of changing something in config and other part of the SIRo codebase. But I do agree with you that the name could be confusing

Reply:
>I am not sure if this is okay as I think it is not legal approved yet
Unless you've heard specific guidance, I think it's safe to use reasonable judgement here. All of SIRo's code is already public OSS. These episode datasets are pretty lightweight data and useless without the accompanying code and scene dataset. I think it would be safe to make it public.

Reply:
>The choice of the name is to reduce the burden of changing something in config and other part of the SIRo codebase.
I agree it's more work for you in this PR.

But consider the alternative: now that you've created the repo with this confusing name, it's harder to fix the name in the future: nobody but you has the permissions to rename/recreate the repo.